src/SpaceTime/2d/Acceleration2.cs

```diff
@@ -96,20 +84,14 @@ public static Acceleration2 In(Direction2 direction, Acceleration u)
         /// <param name="a0">The acceleration vector at t = 0.</param>
         /// <param name="a1">The acceleration vector at t = 1.</param>
         /// <param name="t">The interpolation scalar.</param>
-        public static Acceleration2 Lerp(Acceleration2 a0, Acceleration2 a1, float t)
-        {
-            return a0 + (a1 - a0) * t;
-        }
+        public static Acceleration2 Lerp(Acceleration2 a0, Acceleration2 a1, float t) => a0 + (a1 - a0) * t;
```

Comment:
I went over all files, and it seems very consistent:
- constructing static functions follow constructors
- static properties (Zero, One) follow member properties
- lerp is grouped with relevant instance method
- operators are at the end of the file

I can see an argument for moving the static properties somewhere else, though I would then say the top of the file, not the bottom. If you want me to change that, let me know.